seismicpro/tests/test_gather.py

```diff
@@ -43,9 +54,10 @@ def compare_gathers(first, second, drop_cols=None, check_types=False, same_surve
 
     first_headers = first.headers.reset_index()
     second_headers = second.headers.reset_index()
-    if drop_cols:
-        first_headers.drop(columns=drop_cols, errors="ignore", inplace=True)
-        second_headers.drop(columns=drop_cols, errors="ignore", inplace=True)
+
+    drop_cols = (to_list(drop_cols) + to_list(EXTERNAL_HEADERS)) if drop_cols is not None else EXTERNAL_HEADERS
```

Comment:
First, `compare_gathers` right now is used only in `test_dump_mere`, where you already add `EXTERNAL_HEADERS` to `drop_cols` list in line 85. It seems that after this line EXTERNAL_HEADERS would be duplicated in `drop_cols`. Since this function should just compare two gathers and already has a `drop_cols`  arg, I believe it is better to cook `drop_cols` properly everywhere else and make this function unaware of any external constants.
Second note, regarding the code style only: IMO it would be better to write such an expression like this
```suggestion
    drop_cols = to_list(EXTERNAL_HEADERS) + (to_list(drop_cols) if drop_cols is not None else [])
```

Reply:
done